internal/request/graphql/schema/collection.go

```diff
@@ -345,6 +369,35 @@ func indexFromAST(directive *ast.Directive) (client.IndexDescription, error) {
 	return desc, nil
 }
 
+func defaultFromAST(
+	field *ast.FieldDefinition,
+	directive *ast.Directive,
+) (any, error) {
+	astNamed, ok := field.Type.(*ast.Named)
+	if !ok {
+		return nil, NewErrDefaultValueNotAllowed(field.Name.Value, field.Type.String())
```

Comment:
todo: Please add tests for this error condition

Reply:
done